app/editor/src/features/admin/sources/schedules/ScheduleAdvanced.tsx

```diff
@@ -28,8 +28,20 @@ export const ScheduleAdvanced: React.FC<IScheduleAdvancedProps> = ({ message })
     setIndex(values.schedules.length);
   };
 
-  const handleDone = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
-    setIndex(undefined);
+  const handleDone = async (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    if (!!index) {
+      try {
+        // TODO: Validation doesn't work.
```

Comment:
Can't get this to work correctly.  The validation does 'work' in the sense it catches the errors, but the formik fields never display the errors.